### PR TITLE
Multivm readme changes for media-streaming

### PR DESCRIPTION
**Pull request by komal1204**

### CODE REVIEW

**Comment by neo-apz** [comment]
Please rebase and bring in the latest changes from `dev`. Update the travis file to build images using the new build system.

**Comment by alok123t** [comment]
Do not merge yet, have to rebase again.

File: benchmarks/media-streaming/dataset/Dockerfile [hunk 1]
```diff
@@ -1,18 +1,24 @@
...
 	build-essential \
-	python3.5 \
-	ffmpeg \
-  curl \
+	curl \
+	wget \
+	yasm \
 	&& rm -rf /var/lib/apt/lists/*
 
+RUN wget --progress=bar:force -O /tmp/ffmpeg-4.1.5.tar.bz2 http://ffmpeg.org/releases/ffmpeg-4.1.5.tar.bz2 \
```
Comment (neo-apz) [inline benchmarks/media-streaming/dataset/Dockerfile:13]: Why not get `ffmpeg` using `apt-get`?
Compiling it takes a long time, particularly on arm and risc-v as they are being emulated when building.

Comment (alok123t) [reply benchmarks/media-streaming/dataset/Dockerfile:13]: There is no ffmpeg for riscv, so we have installed it from source instead
https://packages.debian.org/buster/ffmpeg

Comment (neo-apz) [reply benchmarks/media-streaming/dataset/Dockerfile:13]: This makes it impossible to build on travis.

Comment (neo-apz) [reply benchmarks/media-streaming/dataset/Dockerfile:13]: OK so there are two solutions in my opinion, either we drop the risc-v support for this workload for now, or we build using the source only for risc-v. I think you can handle this by using the parameter `$TARGETPLATFORM` inside the dockerfile and decided what to do based on the platform. Can you try this?

Comment (neo-apz) [reply benchmarks/media-streaming/dataset/Dockerfile:13]: (We are going to hit this risc-v wall several other times probably for other workloads, so we need to decide what we can we about it.)

Comment (alok123t) [reply benchmarks/media-streaming/dataset/Dockerfile:13]: I ran the buildx command for dataset on baremetal instance (nproc --all = 48), it takes ~28mins to run for all 3 architectures combined. (Travis has only 2 cores)
Option1: Let the Travis build only for amd64 and we can manually push the image.
Option2: Have conditional install based on the platform
```
RUN set -ex \
	&& case "$TARGETPLATFORM" in \
	# riscv64 debian sid
	linux/riscv64) \
            # install from source
        *) \
            # install using apt-get
```
I guess in both cases we cannot build for risc-v on travis, as it takes more than the travis time limit.

Comment (neo-apz) [reply benchmarks/media-streaming/dataset/Dockerfile:13]: Yeah, I also tried it locally and it took around half an hour to complete, but things were running in parallel. On Travis, the build are done sequentially as there are not enough cores. My hope is that if we only build from source for risc-v, the other two will finish quickly and there is more time for the risc-v build to complete, don't know how long it's gonna take on Travis though.

Here is what I suggest:
* remove risc-v for this benchmark for now so we can merge this and move on
* create another PR that tries to add risc-v based on what we discussed above (perhaps we need to change the time limit to 50, or more if possible)

Comment (neo-apz) [reply benchmarks/media-streaming/dataset/Dockerfile]: You forgot to add ARM.

Comment (alok123t) [reply benchmarks/media-streaming/dataset/Dockerfile]: Updated

File: docs/benchmarks/media-streaming.md [hunk 2]
```diff
@@ -18,46 +18,52 @@ Supported tags and their respective `Dockerfile` links:
...
+Copy logs folder from container to Host1:
 
-### Creating a network between the server and the client(s)
-
-To facilitate the communication between the client(s) and the server, we build a docker network:
-
-    $ docker network create streaming_network
-
-We will attach the launched containers to this newly created docker network.
+    $ docker cp ${DATASET_CONTAINER_ID}:/videos/logs /home/ubuntu/
```
Comment (neo-apz) [inline docs/benchmarks/media-streaming.md:35]: Can we have a better name (path) instead of using `ubuntu` in the path?

**Comment by neo-apz** [comment]
This looks good to me, just address the above comment and we will merge it.

**Pull request merged**